lib/services/groups/groupRegistryMemory.js

```diff
@@ -152,6 +186,32 @@ function findSingleConfigurationMode(service, subservice, callback) {
     }
 }
 
+function findConfigGroups(service, subservice, callback) {
+    if (config.getConfig().singleConfigurationMode === true) {
+        return findSingleConfigurationMode(service, subservice, callback);
+    }
+    const result = [];
+
+    for (const i in registeredGroups) {
+        if (
+            registeredGroups.hasOwnProperty(i) &&
+            registeredGroups[i].service === service &&
+            registeredGroups[i].subservice === subservice
+        ) {
+            result.push(registeredGroups[i]);
+        }
+    }
+
+    if (result.length > 0) {
+        return callback(null, {
+            count: result.length,
+            configGroups: result
+        });
+    } else {
+        return callback(new errors.DeviceGroupNotFound(service, subservice));
+    }
+}
+
```

Comment:
This is duplicated, same as `function find(service, subservice, callback)`.  No needed to duplicate code

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11